src/dcurl.c

```diff
@@ -96,6 +113,30 @@ int8_t *dcurl_entry(int8_t *trytes, int mwm, int threads)
     if (!isInitialized)
         return NULL;
 
+#if defined(ENABLE_REMOTE)
+    void *pow_remote_ctx = NULL;
+
+    do {
+        if (enterRemoteContext(&Remote_Context)) {
+            pow_remote_ctx = getRemoteContext(&Remote_Context, trytes, mwm);
+            goto do_remote;
+        }
+        uv_sem_wait(&notify_remote);
+    } while (1);
+
+remote_pow:
+    if (!doRemoteContext(&Remote_Context, pow_remote_ctx)) {
```

Comment:
It is ambiguous to figure out two variables with similar names: `Remote_Context` and `pow_remote_ctx`.
Can you simply say `remote_ctx` and `pow_ctx`? The later is just a local variable.

Reply:
I rename `Remote_Context` to `Remote_ImplConext`. In the future, we maybe use ZeroMQ to Implement RPC or use others and replace `pw_remote_ctx` with `pow_ctx`.

Reply:
It sounds good.